community/tlp/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+
+# This is needed since tlp uses the non posix compliant -w flag of flock
```

Comment:
Full stop needed. Everything else is great. Thanks for making the changes. :+1:

Reply:
Don't worry about the commit history, I'll squash it all during the merge.

Reply:
What does full stop mean in this context? (I'm not a native speaker, sorry if it's a silly question)

Reply:
Ah. Apologies. 

```
 # This is needed since tlp uses the non posix compliant -w flag of flock
```

becomes

```
 # This is needed since tlp uses the non posix compliant -w flag of flock.
```

Reply:
Notice the `.` on the end. :+1:

Reply:
nevermind, it's punctuation, got it.